src/store/api/ContactUsAPI.ts

```diff
@@ -0,0 +1,22 @@
+import axios from "axios";
+import requestConfig from "./config";
+import { IContactUs } from "../../interfaces";
+const BASE_URL = process.env.REACT_APP_API_ENDPOINT as string;
+
+class ContactUsAPI{
+
+    static getselectedContactUsId(contactUsId : string): Promise <IContactUs> {
```

Comment:
Change `getselectedContactUsId` to `getSelectedContactUsById` or `getSelectedContactUs`. Something like that. Because the function will return the selected contact. not an Id.